src/student.js

```diff
@@ -0,0 +1,8 @@
+export default class Student {
+  constructor(name, id, githubUsername, email) {
+    ;[this.firstName, this.lastName] = name.split(' ')
```

Comment:
Interesting way to create a full name! Does the semi-colon at the beginning do anything?